From 5ba1592215a7a1f20e6586ec7a23c9bc97dd63c4 Mon Sep 17 00:00:00 2001
From: Gurchetan Singh <gurchetansingh@chromium.org>
Date: Thu, 28 Mar 2019 20:51:39 -0700
Subject: [PATCH] HACK: disable VK_KHR_draw_indirect_count

Fixes:

dEQP-VK.draw.indirect_draw.indexed.indirect_draw_count#triangle_list
dEQP-VK.draw.indirect_draw.indexed.indirect_draw_count#triangle_strip
dEQP-VK.draw.indirect_draw.indexed.indirect_draw_count_first_instance#triangle_list
dEQP-VK.draw.indirect_draw.indexed.indirect_draw_count_first_instance#triangle_strip

Change-Id: I3ee73763aec5da085d2b4c92293080e419846c67
---
 src/intel/vulkan/anv_extensions.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/intel/vulkan/anv_extensions.py b/src/intel/vulkan/anv_extensions.py
index 5853b8848f7f..d8044a56b63f 100644
--- a/src/intel/vulkan/anv_extensions.py
+++ b/src/intel/vulkan/anv_extensions.py
@@ -81,7 +81,7 @@ EXTENSIONS = [
     Extension('VK_KHR_device_group',                      1, True),
     Extension('VK_KHR_device_group_creation',             1, True),
     Extension('VK_KHR_display',                          23, 'VK_USE_PLATFORM_DISPLAY_KHR'),
-    Extension('VK_KHR_draw_indirect_count',               1, True),
+    Extension('VK_KHR_draw_indirect_count',               1, False),
     Extension('VK_KHR_driver_properties',                 1, False),
     Extension('VK_KHR_external_fence',                    1,
               'device->has_syncobj_wait'),
-- 
2.20.1

